openapi.yaml

```diff
@@ -7697,6 +7699,18 @@ components:
         - tool
         - function
 
+    ChatCompletionStreamOptions:
+      description: |
+        Options for streaming response. Only set this when you set `stream: true`.
+      type: object
+      nullable: true
+      default: null
```

Comment:
This is interesting, I wonder if they meant this to be for the `include_usage` property. AFAIK this doesn't have any effect on the Model

Reply:
Maybe I'm wrong